File: src/main/java/com/catan/catanbackend/model/dto/TileDto.java
package com.catan.catanbackend.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TileDto {
    private Integer x;
    private Integer y;
    private Integer z;
    private String tileType;
    private Integer number;
    private Boolean hasRobber;
}

File: Assets/Scripts/Dtos/TileDto.cs
using System;

namespace Assets.Scripts.Dtos
{
    [Serializable]
    public class TileDto
    {
        public int x;
        public int y;
        public int z=0;
        public string tileType;
        public int number;

        public TileDto(int x, int y, string tileType, int number,int z)
        {
            this.x = x;
            this.y = y;
            this.z = z;
            this.tileType = tileType;
            this.number = number;
        }
    }
}
